assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
+            "rwy": {
+                "KMIA8L": ["BEETR", "#053"],
+                "KMIA8R": ["BEETR", "#053"],
+                "KMIA9": ["BEETR", "#053"],
+                "KMIA12": ["BEETR", "#053"],
+                "KMIA26L": ["LECIT", "#124"],
+                "KMIA26R": ["LECIT", "#124"],
+                "KMIA27": ["LECIT", "#124"],
+                "KMIA30": ["LECIT", "#124"]
             },
-            "body": [["CURSO", "A100|S250"], "MNNDY", "JODPO", "BEETR"],
-            "draw": [["EADEN*","STRAP","TIGAR","CARNU","DVALL"],["MTH*","GUSHY","DVALL"],["DVALL","CURSO","MNNDY","JODPO","BEETR"]]
+            "draw": [
+                ["EADEN*", "STRAP", "TIGAR", "CARNU", "DVALL", "CURSO", "MNNDY", "JODPO", "BEETR"],
+                ["MTH*", "GUSHY", "DVALL"],
+                ["JODPO", "LECIT"]
+            ]
         },
-        "CYY7": {
-            "icao": "CYY7",
-            "name": "Cypress Seven",
+        "CYY8": {
+            "icao": "CYY8",
+            "name": "Cypress Eight",
             "entryPoints": {
-                "SRQ": ["SRQ", "CYY"],
-                "PIE": ["PIE", "CYY"],
-                "RSW": ["RSW", "CYY"]
+                "PIE": ["PIE"],
+                "RSW": ["RSW"],
+                "SRQ": ["SRQ"]
             },
-            "body": ["CYY", ["DEEDS", "A160"], ["WORPP", "A100|S250"], "DHP"],
-            "draw": [["SRQ*", "CYY"], ["PIE*", "CYY"], ["RSW*", "CYY"], ["CYY", "DEEDS", "WORPP", "DHP"]]
+            "body": ["CYY", "DEEDS", "WORPP", "DHP"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["PIE*", "CYY", "DEEDS", "WORPP", "DHP"],
+                ["RSW*", "CYY"],
+                ["SRQ*", "CYY"]
+            ]
         },
         "DVALL2": {
             "icao": "DVALL2",
-            "name": "Dvall Two",
+            "name": "DaVall Two",
             "entryPoints": {
-                "EYW": ["EYW", "CARNU", "DVALL"],
-                "MTH": ["MTH", "DVALL"]
+                "EYW": ["EYW", "CARNU"],
+                "MTH": ["MTH"]
             },
-            "body": [["WEVER", "A100|S250"], "FAMIN", "DHP"],
-            "draw": [["EYW*","CARNU","DVALL"],["MTH*","DVALL"],["DVALL","WEVER","FAMIN","DHP"]]
+            "body": ["DVALL", "WEVER", "FAMIN", "DHP"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["EYW*", "CARNU", "DVALL", "WEVER", "FAMIN", "DHP"],
+                ["MTH*", "DVALL"]
+            ]
         },
         "FLIPR4": {
```

Comment:
FLIPR4 is now FLIPR5

Reply:
reverified and updated all SIDs/STARs.

done